packages/core/src/reducers/connectivityStatus.js

```diff
@@ -12,6 +12,8 @@ import {
 
 import { DISCONNECT_FULFILLED } from '../../lib/actions/disconnect';
```

Comment:
Not your fault. But, I believe this should no go all the way up to `../../lib/actions/disconnect`. Should be `../actions/disconnect`. We can fix this later.